### PR TITLE
adds man page for Turbo

### PR DESCRIPTION
**Pull request by amay5267**
Nothing. Just MAN.

### CODE REVIEW

File: man/Makefile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+PAGE_SRCS := $(shell find -name '*.[0-9].txt' -printf '%P\n')
+PAGES := $(patsubst %.txt,output/%,$(PAGE_SRCS))
+
+all: $(PAGES)
+
+output/%: %.txt
+	@mkdir -p $$(dirname '$@')
+	@echo [GEN] $@ ; manscript -o $@ $<
```
Comment (waebbl) [inline man/Makefile:8]: Where does this `manscript` come from? Never heard about it.

Comment (amay5267) [reply man/Makefile:8]: https://github.com/gphalkes/tilde/tree/master/man

Comment (amay5267) [reply man/Makefile]: Maybe we should use old school turbo.1.in https://github.com/amay5267/turbo/blob/amay5267-patch-1/man/turbo.1.in

Comment (amay5267) [reply man/Makefile]: Can you please write proper cross-platform Makefile for this MAN?
I can test and review that on macOS as well.

**Comment by magiblot** [comment]
Hi @amay5267! Thanks for the suggestion and for the pull request. However, I finally wrote a new manpage from scratch.

**Pull request closed without merging**